src/types/controlPlaneConfig.ts

```diff
@@ -19,6 +42,8 @@ export type Destination<DestinationConfig = Record<string, unknown>> = {
   RevisionID?: string;
   IsProcessorEnabled?: boolean;
   IsConnectionEnabled?: boolean;
+  DeliveryAccount?: DeliveryAccountT;
```

Comment:
is it DeliveryAccount or deliveryAccount?
is it DeleteAccount or deleteAccount?
We are using json struct tags right so did you check how they are coming to transformer
https://github.com/rudderlabs/rudder-server/blob/89bc62bd9ab21c546be4b9acb7f94c5f240a3ef3/backend-config/types.go#L61

Reply:
It is deliveryAccount and deleteAccount. Updated the code.